plugins/inputs/slurm/sample.conf

```diff
@@ -0,0 +1,28 @@
+# Gather SLURM metrics
+[[inputs.slurm]]
+  ## Slurmrestd URL. Both http and https can be used as schemas.
+  url = "http://127.0.0.1:6820"
+
+  ## Credentials for JWT-based authentication.
+  # username = "foo"
+  # token = "topSecret"
+
+  ## Ignored endpoints
+  ## List of endpoints a user can ignore, choose from: diag, jobs,
+  ##   nodes, partitions, reservations
+  ## Please note incorrect endpoints will be silently ignore and
+  ## that endpoint names are case insensitive.
+  # ignored_endpoints = []
+
+  ## Maximum time to receive response.
+  # response_timeout = "5s"
+
+  ## Optional TLS Config. Note these options will only
+  ## be taken into account when the scheme specififed on
+  ## the URL parameter is https. They will be silently
+  ## ignored otherwise.
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+  ## Use TLS but skip chain & host verification
+  # insecure_skip_verify = false
```

Comment:
How about automatically include all possible TLS options? See the [inputs.http sample.conf.in file](https://github.com/influxdata/telegraf/blob/ad55f42d5366eb57e46e9232a637a84e8c23e9a5/plugins/inputs/http/sample.conf.in#L42) for an example. If you do this, rename the `sample.conf` to `sample.conf.in`, add the template and include the [sample.conf generator](https://github.com/influxdata/telegraf/blob/ad55f42d5366eb57e46e9232a637a84e8c23e9a5/plugins/inputs/http/http.go#L1) to the `slurm.go` file.

Reply:
Thanks a ton for the suggestion! I added the changes on 2da1947040cdba0a664da15e5663d3ea0b604b4e.